2020/08/26 21:05:35 [error] 5132#2212333: *16 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:05:35 [error] 5132#2212333: *17 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:08:52 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:08:52 [notice] 5131#2212332: reconfiguring
2020/08/26 21:08:52 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:08:52 [notice] 5131#2212332: start worker processes
2020/08/26 21:08:52 [notice] 5131#2212332: start worker process 5368
2020/08/26 21:08:52 [error] 5368#2214706: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: API disabled in the context of init_worker_by_lua*
stack traceback:
	[C]: in function 'say'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:46: in main chunk
2020/08/26 21:08:52 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:08:52 [notice] 5132#2212333: gracefully shutting down
2020/08/26 21:08:52 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:08:52 [notice] 5132#2212333: exiting
2020/08/26 21:08:52 [notice] 5132#2212333: exit
2020/08/26 21:08:52 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5132
2020/08/26 21:08:52 [notice] 5131#2212332: worker process 5132 exited with code 0
2020/08/26 21:08:52 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:08:52 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:09:06 [error] 5368#2214706: *19 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:06 [error] 5368#2214706: *20 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:06 [error] 5368#2214706: *21 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:06 [error] 5368#2214706: *22 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:07 [error] 5368#2214706: *23 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:07 [error] 5368#2214706: *24 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:07 [error] 5368#2214706: *25 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:07 [error] 5368#2214706: *26 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:07 [error] 5368#2214706: *27 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:07 [error] 5368#2214706: *28 failed to run balancer_by_lua*: balancer_by_lua:3: module 'lua/upstreams.lua' not found:
	no field package.preload['lua/upstreams.lua']
	no file '/Users/4paradigm/Desktop/monitor-proxy/lua/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.ljbc'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua/init.lua'
	no file './lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/luajit-2.1.0-beta3/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/share/lua/5.1/lua/upstreams/lua/init.lua'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams/lua.so'
	no file './lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams/lua.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/site/lualib/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/lualib/lua/upstreams.so'
	no file './lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/Cellar/openresty/1.17.8.2_1/luajit/lib/lua/5.1/lua/upstreams.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:48 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:09:48 [notice] 5131#2212332: reconfiguring
2020/08/26 21:09:48 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:09:48 [notice] 5131#2212332: start worker processes
2020/08/26 21:09:48 [notice] 5131#2212332: start worker process 5509
2020/08/26 21:09:48 [error] 5509#2215536: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: API disabled in the context of init_worker_by_lua*
stack traceback:
	[C]: in function 'say'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:46: in main chunk
2020/08/26 21:09:49 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:09:49 [notice] 5368#2214706: gracefully shutting down
2020/08/26 21:09:49 [notice] 5368#2214706: exiting
2020/08/26 21:09:49 [notice] 5131#2212332: signal 23 (SIGIO) received from 5368
2020/08/26 21:09:49 [notice] 5368#2214706: exit
2020/08/26 21:09:49 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5368
2020/08/26 21:09:49 [notice] 5131#2212332: worker process 5368 exited with code 0
2020/08/26 21:09:49 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:09:49 [notice] 5131#2212332: signal 23 (SIGIO) received from 5509
2020/08/26 21:09:56 [error] 5509#2215536: *30 failed to run balancer_by_lua*: ./lua/upstreams.lua:19: API disabled in the context of balancer_by_lua*
stack traceback:
	[C]: in function 'say'
	./lua/upstreams.lua:19: in function 'init_getstreams'
	./lua/upstreams.lua:46: in main chunk
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:56 [error] 5509#2215536: *31 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:09:57 [error] 5509#2215536: *32 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:09:57 [error] 5509#2215536: *33 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:10:12 [error] 5509#2215536: *34 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:10:12 [error] 5509#2215536: *35 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:10:32 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:10:32 [notice] 5131#2212332: reconfiguring
2020/08/26 21:10:32 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:10:32 [notice] 5131#2212332: start worker processes
2020/08/26 21:10:32 [notice] 5131#2212332: start worker process 5608
2020/08/26 21:10:32 [error] 5608#2216246: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: API disabled in the context of init_worker_by_lua*
stack traceback:
	[C]: in function 'say'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:19: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:46: in main chunk
2020/08/26 21:10:32 [notice] 5509#2215536: gracefully shutting down
2020/08/26 21:10:32 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:10:32 [notice] 5509#2215536: exiting
2020/08/26 21:10:32 [notice] 5509#2215536: exit
2020/08/26 21:10:32 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5509
2020/08/26 21:10:32 [notice] 5131#2212332: worker process 5509 exited with code 0
2020/08/26 21:10:32 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:10:32 [notice] 5131#2212332: signal 23 (SIGIO) received from 5608
2020/08/26 21:10:32 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:11:00 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:11:00 [notice] 5131#2212332: reconfiguring
2020/08/26 21:11:00 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:11:00 [notice] 5131#2212332: start worker processes
2020/08/26 21:11:00 [notice] 5131#2212332: start worker process 5700
2020/08/26 21:11:00 [info] 5700#2216789: *37 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:11:00 [error] 5700#2216789: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: attempt to index field 'upstream_list' (a nil value)
stack traceback:
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:45: in main chunk
2020/08/26 21:11:00 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:11:00 [notice] 5131#2212332: signal 23 (SIGIO) received from 5608
2020/08/26 21:11:00 [notice] 5608#2216246: gracefully shutting down
2020/08/26 21:11:00 [notice] 5608#2216246: exiting
2020/08/26 21:11:00 [notice] 5608#2216246: exit
2020/08/26 21:11:00 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5608
2020/08/26 21:11:00 [notice] 5131#2212332: worker process 5608 exited with code 0
2020/08/26 21:11:00 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:11:00 [notice] 5131#2212332: signal 23 (SIGIO) received from 5700
2020/08/26 21:11:00 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:11:34 [info] 5700#2216789: *38 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:11:34 [error] 5700#2216789: *38 failed to run balancer_by_lua*: ./lua/upstreams.lua:35: attempt to index field 'upstream_list' (a nil value)
stack traceback:
	./lua/upstreams.lua:35: in function 'init_getstreams'
	./lua/upstreams.lua:45: in main chunk
	[C]: in function 'require'
	balancer_by_lua:3: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:11:34 [error] 5700#2216789: *39 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:11:35 [error] 5700#2216789: *40 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:11:35 [error] 5700#2216789: *41 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:13:16 [error] 5700#2216789: *42 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:13:17 [error] 5700#2216789: *43 failed to run balancer_by_lua*: balancer_by_lua:5: attempt to index local 'upstreams' (a userdata value)
stack traceback:
	balancer_by_lua:5: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2020/08/26 21:21:41 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:21:41 [notice] 5131#2212332: reconfiguring
2020/08/26 21:21:41 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:21:41 [notice] 5131#2212332: start worker processes
2020/08/26 21:21:41 [notice] 5131#2212332: start worker process 5925
2020/08/26 21:21:41 [info] 5925#2221893: *44 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:21:41 [error] 5925#2221893: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: attempt to index field 'upstream_list' (a nil value)
stack traceback:
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:45: in main chunk
2020/08/26 21:21:41 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:21:41 [notice] 5131#2212332: signal 23 (SIGIO) received from 5700
2020/08/26 21:21:41 [notice] 5700#2216789: gracefully shutting down
2020/08/26 21:21:41 [notice] 5700#2216789: exiting
2020/08/26 21:21:41 [notice] 5700#2216789: exit
2020/08/26 21:21:41 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5700
2020/08/26 21:21:41 [notice] 5131#2212332: worker process 5700 exited with code 0
2020/08/26 21:21:41 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:21:41 [notice] 5131#2212332: signal 23 (SIGIO) received from 5925
2020/08/26 21:21:45 [error] 5925#2221893: *45 connect() to 0.0.0.1:80 failed (65: No route to host) while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", upstream: "http://0.0.0.1:80/", host: "localhost"
2020/08/26 21:21:45 [warn] 5925#2221893: *45 upstream server temporarily disabled while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", upstream: "http://0.0.0.1:80/", host: "localhost"
2020/08/26 21:22:45 [info] 5925#2221893: *46 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:24:34 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:24:34 [notice] 5131#2212332: reconfiguring
2020/08/26 21:24:34 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:24:34 [notice] 5131#2212332: start worker processes
2020/08/26 21:24:34 [notice] 5131#2212332: start worker process 6052
2020/08/26 21:24:34 [info] 6052#2223673: *62 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:24:34 [error] 6052#2223673: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: attempt to index field 'upstream_list' (a nil value)
stack traceback:
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:45: in main chunk
2020/08/26 21:24:34 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:34 [notice] 5925#2221893: gracefully shutting down
2020/08/26 21:24:34 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:34 [notice] 5925#2221893: exiting
2020/08/26 21:24:34 [notice] 5925#2221893: exit
2020/08/26 21:24:34 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 5925
2020/08/26 21:24:34 [notice] 5131#2212332: worker process 5925 exited with code 0
2020/08/26 21:24:34 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:34 [notice] 5131#2212332: signal 23 (SIGIO) received from 6052
2020/08/26 21:24:46 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:24:46 [notice] 5131#2212332: reconfiguring
2020/08/26 21:24:46 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:24:46 [notice] 5131#2212332: start worker processes
2020/08/26 21:24:46 [notice] 5131#2212332: start worker process 6125
2020/08/26 21:24:46 [info] 6125#2224066: *63 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:24:46 [error] 6125#2224066: init_worker_by_lua_file error: /Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: attempt to index field 'upstream_list' (a nil value)
stack traceback:
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:35: in function 'init_getstreams'
	/Users/4paradigm/Desktop/monitor-proxy/lua/upstreams.lua:45: in main chunk
2020/08/26 21:24:46 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:46 [notice] 6052#2223673: gracefully shutting down
2020/08/26 21:24:46 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:46 [notice] 6052#2223673: exiting
2020/08/26 21:24:46 [notice] 6052#2223673: exit
2020/08/26 21:24:46 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6052
2020/08/26 21:24:46 [notice] 5131#2212332: worker process 6052 exited with code 0
2020/08/26 21:24:46 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:24:46 [notice] 5131#2212332: signal 23 (SIGIO) received from 6125
2020/08/26 21:25:20 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:25:20 [notice] 5131#2212332: reconfiguring
2020/08/26 21:25:20 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:25:20 [notice] 5131#2212332: start worker processes
2020/08/26 21:25:20 [notice] 5131#2212332: start worker process 6229
2020/08/26 21:25:20 [info] 6229#2224713: *64 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:25:21 [notice] 6125#2224066: gracefully shutting down
2020/08/26 21:25:21 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:25:21 [notice] 6125#2224066: exiting
2020/08/26 21:25:21 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:25:21 [notice] 6125#2224066: exit
2020/08/26 21:25:21 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6125
2020/08/26 21:25:21 [notice] 5131#2212332: worker process 6125 exited with code 0
2020/08/26 21:25:21 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:25:21 [notice] 5131#2212332: signal 23 (SIGIO) received from 6229
2020/08/26 21:25:21 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:25:26 [info] 6229#2224713: *65 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:25:26 [error] 6229#2224713: *65 failed to run balancer_by_lua*: balancer_by_lua:6: bad argument #1 to 'getn' (table expected, got string)
stack traceback:
	[builtin:getn]: in function 'getn'
	balancer_by_lua:6: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:25:27 [error] 6229#2224713: *66 failed to run balancer_by_lua*: balancer_by_lua:6: bad argument #1 to 'getn' (table expected, got string)
stack traceback:
	[builtin:getn]: in function 'getn'
	balancer_by_lua:6: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:26:27 [info] 6229#2224713: *67 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:27:02 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:27:02 [notice] 5131#2212332: reconfiguring
2020/08/26 21:27:02 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:27:02 [notice] 5131#2212332: start worker processes
2020/08/26 21:27:02 [notice] 5131#2212332: start worker process 6335
2020/08/26 21:27:02 [info] 6335#2225737: *68 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:27:03 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:27:03 [notice] 6229#2224713: gracefully shutting down
2020/08/26 21:27:03 [notice] 6229#2224713: exiting
2020/08/26 21:27:03 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:27:03 [notice] 6229#2224713: exit
2020/08/26 21:27:03 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6229
2020/08/26 21:27:03 [notice] 5131#2212332: worker process 6229 exited with code 0
2020/08/26 21:27:03 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:27:03 [notice] 5131#2212332: signal 23 (SIGIO) received from 6335
2020/08/26 21:27:07 [info] 6335#2225737: *69 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:27:07 [error] 6335#2225737: *69 failed to run balancer_by_lua*: balancer_by_lua:6: bad argument #1 to 'getn' (table expected, got string)
stack traceback:
	[builtin:getn]: in function 'getn'
	balancer_by_lua:6: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:27:56 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:27:56 [notice] 5131#2212332: reconfiguring
2020/08/26 21:27:56 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:27:56 [notice] 5131#2212332: start worker processes
2020/08/26 21:27:56 [notice] 5131#2212332: start worker process 6412
2020/08/26 21:27:56 [info] 6412#2226361: *71 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:27:56 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:27:56 [notice] 6335#2225737: gracefully shutting down
2020/08/26 21:27:56 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:28:07 [info] 6335#2225737: *70 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:28:07 [notice] 6335#2225737: exiting
2020/08/26 21:28:07 [notice] 6335#2225737: exit
2020/08/26 21:28:07 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6335
2020/08/26 21:28:07 [notice] 5131#2212332: worker process 6335 exited with code 0
2020/08/26 21:28:07 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:28:07 [notice] 5131#2212332: signal 23 (SIGIO) received from 6412
2020/08/26 21:28:21 [error] 6412#2226361: *72 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:28:21 [error] 6412#2226361: *72 [lua] balancer_by_lua:4: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:28:21 [info] 6412#2226361: *72 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:28:21 [error] 6412#2226361: *72 [lua] balancer_by_lua:6: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:28:21 [error] 6412#2226361: *72 [lua] balancer_by_lua:8: ding log 4 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:28:21 [error] 6412#2226361: *72 failed to run balancer_by_lua*: balancer_by_lua:9: bad argument #1 to 'getn' (table expected, got string)
stack traceback:
	[builtin:getn]: in function 'getn'
	balancer_by_lua:9: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:29:21 [info] 6412#2226361: *73 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:35:39 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:35:39 [notice] 5131#2212332: reconfiguring
2020/08/26 21:35:39 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:35:39 [notice] 5131#2212332: start worker processes
2020/08/26 21:35:39 [notice] 5131#2212332: start worker process 6641
2020/08/26 21:35:39 [info] 6641#2230380: *74 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:35:39 [notice] 6412#2226361: gracefully shutting down
2020/08/26 21:35:39 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:35:39 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:35:39 [notice] 6412#2226361: exiting
2020/08/26 21:35:39 [notice] 6412#2226361: exit
2020/08/26 21:35:39 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6412
2020/08/26 21:35:39 [notice] 5131#2212332: worker process 6412 exited with code 0
2020/08/26 21:35:39 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:35:44 [error] 6641#2230380: *75 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:35:44 [error] 6641#2230380: *75 [lua] balancer_by_lua:4: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:35:44 [info] 6641#2230380: *75 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:35:44 [error] 6641#2230380: *75 [lua] balancer_by_lua:6: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:35:44 [error] 6641#2230380: *75 [lua] balancer_by_lua:8: ding log 4string while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:35:44 [error] 6641#2230380: *75 failed to run balancer_by_lua*: balancer_by_lua:9: bad argument #1 to 'getn' (table expected, got string)
stack traceback:
	[builtin:getn]: in function 'getn'
	balancer_by_lua:9: in main chunk while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:36:44 [info] 6641#2230380: *76 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:38:06 [notice] 5131#2212332: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/26 21:38:06 [notice] 5131#2212332: reconfiguring
2020/08/26 21:38:06 [notice] 5131#2212332: using the "kqueue" event method
2020/08/26 21:38:06 [notice] 5131#2212332: start worker processes
2020/08/26 21:38:06 [notice] 5131#2212332: start worker process 6774
2020/08/26 21:38:06 [info] 6774#2231729: *77 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/26 21:38:07 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:38:07 [notice] 6641#2230380: gracefully shutting down
2020/08/26 21:38:07 [notice] 6641#2230380: exiting
2020/08/26 21:38:07 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:38:07 [notice] 6641#2230380: exit
2020/08/26 21:38:07 [notice] 5131#2212332: signal 20 (SIGCHLD) received from 6641
2020/08/26 21:38:07 [notice] 5131#2212332: worker process 6641 exited with code 0
2020/08/26 21:38:07 [notice] 5131#2212332: signal 23 (SIGIO) received
2020/08/26 21:38:07 [notice] 5131#2212332: signal 23 (SIGIO) received from 6774
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [info] 6774#2231729: *78 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:38:09 [error] 6774#2231729: *78 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:39:09 [info] 6774#2231729: *79 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:22 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============4 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============4 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:23 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============6 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:24 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:25 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:27 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============4 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============6 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:28 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:29 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:30 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:2: ding log 1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:5: ding log 2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:7: ding log 3 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:9: ding log 4[{"ip":"127.0.0.1","port":"3344"},{"ip":"127.0.0.1","port":"4455"},{"ip":"127.0.0.1","port":"5566"},{"ip":"127.0.0.1","port":"6677"},{"ip":"127.0.0.1","port":"7788"},{"ip":"127.0.0.1","port":"8899"}] while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:11: ding test 5127.0.0.1 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:13: ding test 5 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:45:32 [error] 6774#2231729: *81 [lua] balancer_by_lua:14: current :=============2 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/26 21:46:22 [info] 6774#2231729: *82 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/27 13:55:43 [notice] 12018#2332009: using the "kqueue" event method
2020/08/27 13:55:43 [notice] 12018#2332009: openresty/1.17.8.2
2020/08/27 13:55:43 [notice] 12018#2332009: built by clang 11.0.3 (clang-1103.0.32.29)
2020/08/27 13:55:43 [notice] 12018#2332009: OS: Darwin 19.0.0
2020/08/27 13:55:43 [notice] 12018#2332009: hw.ncpu: 4
2020/08/27 13:55:43 [notice] 12018#2332009: net.inet.tcp.sendspace: 131072
2020/08/27 13:55:43 [notice] 12018#2332009: kern.ipc.somaxconn: 128
2020/08/27 13:55:43 [notice] 12018#2332009: getrlimit(RLIMIT_NOFILE): 10240:9223372036854775807
2020/08/27 13:55:43 [notice] 12019#2332017: start worker processes
2020/08/27 13:55:43 [notice] 12019#2332017: start worker process 12020
2020/08/27 13:55:43 [info] 12020#2332018: *1 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/27 13:55:47 [error] 12020#2332018: *2 failed to load inlined Lua code: balancer_by_lua:8: unexpected symbol near '#' while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/27 13:55:50 [error] 12020#2332018: *3 failed to load inlined Lua code: balancer_by_lua:8: unexpected symbol near '#' while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/27 13:56:07 [error] 12020#2332018: *4 failed to load inlined Lua code: balancer_by_lua:8: unexpected symbol near '#' while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/27 13:57:07 [info] 12020#2332018: *5 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
2020/08/27 14:01:30 [notice] 12019#2332017: signal 1 (SIGHUP) received from 90263, reconfiguring
2020/08/27 14:01:30 [notice] 12019#2332017: reconfiguring
2020/08/27 14:01:30 [notice] 12019#2332017: using the "kqueue" event method
2020/08/27 14:01:30 [notice] 12019#2332017: start worker processes
2020/08/27 14:01:30 [notice] 12019#2332017: start worker process 12175
2020/08/27 14:01:30 [info] 12175#2335551: *6 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899, context: init_worker_by_lua*
2020/08/27 14:01:30 [notice] 12019#2332017: signal 23 (SIGIO) received
2020/08/27 14:01:30 [notice] 12020#2332018: gracefully shutting down
2020/08/27 14:01:30 [notice] 12019#2332017: signal 23 (SIGIO) received
2020/08/27 14:01:30 [notice] 12020#2332018: exiting
2020/08/27 14:01:30 [notice] 12020#2332018: exit
2020/08/27 14:01:30 [notice] 12019#2332017: signal 20 (SIGCHLD) received from 12020
2020/08/27 14:01:30 [notice] 12019#2332017: worker process 12020 exited with code 0
2020/08/27 14:01:30 [notice] 12019#2332017: signal 23 (SIGIO) received
2020/08/27 14:01:30 [notice] 12019#2332017: signal 23 (SIGIO) received from 12175
2020/08/27 14:01:30 [notice] 12019#2332017: signal 23 (SIGIO) received
2020/08/27 14:02:34 [info] 12175#2335551: *7 [lua] upstreams.lua:20: init_getstreams(): customer defined ab test: 127.0.0.1:3344;127.0.0.1:4455;127.0.0.1:5566;127.0.0.1:6677;127.0.0.1:7788;127.0.0.1:8899 while connecting to upstream, client: 127.0.0.1, server: _, request: "GET / HTTP/1.1", host: "localhost"
2020/08/27 14:03:34 [info] 12175#2335551: *8 client timed out (60: Operation timed out) while waiting for request, client: 127.0.0.1, server: 0.0.0.0:80
